src/util/gobConfig.ml

```diff
@@ -28,6 +28,8 @@ exception ConfigError of string
 (* Phase of the analysis (moved from GoblintUtil b/c of circular build...) *)
 let phase = ref 0
 
+let building_spec = ref false
+
```

Comment:
Is this flag still needed? Where is it read?

Reply:
I replaced the commented out code with tracing below, so now the option has some meaning for now as well.